test/e2e/overrides_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/kyma-project/telemetry-manager/test/testkit/verifiers"
 )
 
-var _ = Describe("Overrides", Label("telemetry"), Ordered, func() {
+var _ = Describe("Overrides", Label("telemetry-overrides"), Ordered, func() {
```

Comment:
Was this change intentional?

Reply:
No, I think it was only for debugging purposes. Sorry and thank you for noticing